### ISSUE TITLE
Cannot unpack game:中出し”以外は校則違反!! ～女子校祭・乗っ取り計画～

### ISSUE THREAD

**Issue by wycstc353** [innocentvoice/EAGLS-Extractor#3]
Cannot unpack game:中出し”以外は校則違反!! ～女子校祭・乗っ取り計画～
http://www.getchu.com/soft.phtml?id=256765&gc=gc

**Comment by innocentvoice** [comment]
would you provide more information?

**Comment by wycstc353** [comment]
![image](https://user-images.githubusercontent.com/39401653/181914661-f4817838-2526-467b-8251-45099bf28dee.png)
有关调用实时(JIT)调试而不是此对话框的详细信息，
请参见此消息的结尾。

************** 异常文本 **************
System.IndexOutOfRangeException: 索引超出了数组界限。
   在 scriptthingy.ScriptFile.TryToFindDataFileKey(Byte[] dataFile, List`1 FileOffsets, Int32 keyLength, Byte[]& outputKey)
   在 scriptthingy.ScriptFile.FindDataFileKey(Byte[] encryptedDataFile, Byte[] decryptedIndexFile)
   在 scriptthingy.MainForm.ExtractScript(String fileName, String exportPath, Boolean useDatFiles)
   在 scriptthingy.MainForm.ExtractScript()
   在 System.Windows.Forms.Control.OnClick(EventArgs e)
   在 System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   在 System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   在 System.Windows.Forms.Control.WndProc(Message& m)
   在 System.Windows.Forms.ButtonBase.WndProc(Message& m)
   在 System.Windows.Forms.Button.WndProc(Message& m)
   在 System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   在 System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** 已加载的程序集 **************
mscorlib
    程序集版本: 2.0.0.0
    Win32 版本: 2.0.50727.9164 (WinRelRS6.050727-9100)
    基本代码: file:///C:/Windows/Microsoft.NET/Framework64/v2.0.50727/mscorlib.dll
----------------------------------------
EAGLS_Decrypt
    程序集版本: 1.0.0.0
    Win32 版本: 1.0.0.0
    基本代码: file:///E:/asdsa/gongju/汉化相关/HSHINTAI_decrypt（EAGLS引擎解包封包工具）/EAGLS_Decrypt.exe
----------------------------------------
System.Windows.Forms
    程序集版本: 2.0.0.0
    Win32 版本: 2.0.50727.9149 (WinRelRS6.050727-9100)
    基本代码: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Windows.Forms/2.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    程序集版本: 2.0.0.0
    Win32 版本: 2.0.50727.9149 (WinRelRS6.050727-9100)
    基本代码: file:///C:/WINDOWS/assembly/GAC_MSIL/System/2.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    程序集版本: 2.0.0.0
    Win32 版本: 2.0.50727.9149 (WinRelRS6.050727-9100)
    基本代码: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Drawing/2.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Core
    程序集版本: 3.5.0.0
    Win32 版本: 3.5.30729.9141 built by: WinRelRS6
    基本代码: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Core/3.5.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
Accessibility
    程序集版本: 2.0.0.0
    Win32 版本: 2.0.50727.9149 (WinRelRS6.050727-9100)
    基本代码: file:///C:/WINDOWS/assembly/GAC_MSIL/Accessibility/2.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
mscorlib.resources
    程序集版本: 2.0.0.0
    Win32 版本: 2.0.50727.9164 (WinRelRS6.050727-9100)
    基本代码: file:///C:/Windows/Microsoft.NET/Framework64/v2.0.50727/mscorlib.dll
----------------------------------------
System.Windows.Forms.resources
    程序集版本: 2.0.0.0
    Win32 版本: 2.0.50727.9149 (WinRelRS6.050727-9100)
    基本代码: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Windows.Forms.resources/2.0.0.0_zh-CHS_b77a5c561934e089/System.Windows.Forms.resources.dll
----------------------------------------

************** JIT 调试 **************
要启用实时(JIT)调试，
该应用程序或计算机的 .config 文件(machine.config)的 system.windows.forms 节中必须设置
jitDebugging 值。
编译应用程序时还必须启用
调试。

例如: 

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

启用 JIT 调试后，任何无法处理的异常
都将被发送到在此计算机上注册的 JIT 调试器，
而不是由此对话框处理。

**Comment by wycstc353** [comment]
> would you provide more information? (e.g errors display on screen, display nothing, program crashed, etc...)

I sent it

**Comment by innocentvoice** [comment]
i have looked at the game few days ago. It got a newer header which the program is unsupported. I might fix the issue or not fixing it. Have you tried Garbro?

**Comment by wycstc353** [comment]
> i have looked at the game few days ago. It got a newer header which the program is unsupported. I might fix the issue or not fixing it. Have you tried Garbro?

Garbro can unpack the game, but he doesn't have a method of packaging, and DAT files have problems such as file headers

**Comment by innocentvoice** [comment]
> > i have looked at the game few days ago. It got a newer header which the program is unsupported. I might fix the issue or not fixing it. Have you tried Garbro?
> 
> Garbro can unpack the game, but he doesn't have a method of packaging, and DAT files have problems such as file headers

该软件专为2014年或更低版本的游戏设计

**Comment by innocentvoice** [comment]
我会发布这个软件的源代码

**Comment by wycstc353** [comment]
> 校則違反

No, the game was released in 2006, but it's strange that the game released by the same company in 2016 can be unpacked

**Comment by wycstc353** [comment]
> 我会发布这个软件的源代码

链接：https://pan.baidu.com/s/1pD97BFvs1wDuPqcQUFsM3A 
提取码：nptx 
--来自百度网盘超级会员V4的分享
This is the script folder of that game
